doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
ROOT_DIR is defined in exactly the same places (2) that ROOT is defined, right below the ROOT definitions. There is no overriding going on.